### PR TITLE
@ashfurrow => [Artwork] Nullify scrollview delegate after all.

### PR DESCRIPTION
**Pull request by alloy**
Fix for https://rink.hockeyapp.net/manage/apps/37029/app_versions/26/crash_reasons/30327711

Related to:

* https://github.com/artsy/eigen/issues/103
* https://github.com/artsy/eigen/pull/218#issuecomment-75958606

### CODE REVIEW

File: docs/BETA_CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ## 2015.03.27
 
+* Nullify scrollview delegate of artview after all to ensure it can never crash again - alloy
```
Comment (ashfurrow) [inline docs/BETA_CHANGELOG.md]: oic

**Comment by ashfurrow** [comment]
Is this a change we could/should test?

**Comment by orta** [comment]
not sure I can think of a way to test this, personally.

**Comment by Inferis** [comment]
(assign) delegates are such a pest.

**Comment by alloy** [comment]
@Inferis Yeah :( Please fix UIScrollView soon ;)

**Pull request merged**